src/deepsparse/loggers/build_logger.py

```diff
@@ -67,32 +76,56 @@ def default_logger() -> Dict[str, BaseLogger]:
     return {"python": PythonLogger()}
 
 
-def build_logger(server_config: ServerConfig) -> BaseLogger:
+def logger_from_config(config: Union[PipelineLoggingConfig, str]) -> BaseLogger:
     """
-    Builds a DeepSparse logger from the ServerConfig.
-
-    The process follows the following hierarchy:
-
-    First: if global logger config is provided,
-        the "leaf" loggers are built.
-
-    Second: if data logging config is specified, a set of
-        function loggers wraps around the appropriate "leaf" loggers.
+     Builds a pipeline logger from the ServerConfig
+
+    :param config: The configuration of the pipeline logger.
+        Can be:
+         - a string (path to the .yaml file)
+         - a string (string representation of the .yaml file)
+         - a PipelineLoggingConfig instance
+    :return: The pipeline with the logger added
+    """
+    if not isinstance(config, str):
+        pass
```

Comment:
This is just converting the str possibility into a config, it could just be:
```python
if isinstance(config, str):
    ...
    config = PipelineLoggingConfig(...)
```